apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -46,17 +47,24 @@ export class AITagView extends React.Component {
   }
 
   validateImage = async () => {
+    const FILE_LIMIT = 5000000; // 5MB limit from AWS Rekognition
+    const FILE_DIMENSION_LIMIT = 80; // 80px limit
     const imageId = get(this.props.entries.image.getValue(), 'sys.id');
-    if (!imageId) { return true; }
+    if (!imageId) { return ; }
 
     const file = await this.props.space.getAsset(imageId);
     const locale = this.props.locale;
     const contentType = get(file, `fields.file.${locale}.contentType`);
+    const details = get(file, `fields.file.${locale}.details`);
     // test if file extension is PNG/JPEG/JPG
-    const isImageValid = new RegExp(/image\/png|jpeg|jpg$/, 'i').test(contentType);
+    const isImageTypeValid = new RegExp(/image\/png|jpeg|jpg$/, 'i').test(contentType);
```

Comment:
I think this regex is missing a capture group. I'd do `/^image\/(png|jpe?g)$/i`

Reply:
why do we need a capture group?